docs/groovy-script/mods/selectionguicrafting/category.md

```diff
@@ -0,0 +1,187 @@
+source_code_link: "https://github.com/Ender-Development/selection-gui-crafting-continued/blob/master/src/main/java/io/enderdev/selectionguicrafting/integration/groovyscript/Category.java"
+---
+
+# Crafting Category (Selection GUI Crafting)
+
+## Description
+
+All recipes in the mod are divided into categories. Each category has its own set of recipes. Each category can have its own texture for the background, border, frame, decoration, and progress bar. The category can also have its own sounds and particles. It also allows to specify how the sounds are played, if recipes in the category can be added to the crafting queue and how the output items are handed to the player.
+
+## Identifier
+
+Refer to this via any of the following:
+
+```groovy:no-line-numbers {1}
+mods.selectionguicrafting.category/* Used as page default */ // [!code focus]
+mods.selectionguicrafting.Category
+```
+
+
+## Adding Recipes
+
+### Recipe Builder
+
+Just like other recipe types, the Crafting Category also uses a recipe builder.
+
+Don't know what a builder is? Check [the builder info page](../../getting_started/builder.md) out.
+
+:::::::::: details mods.selectionguicrafting.category.categoryBuilder() {open id="abstract"}
+- `String`. The Category ID, must be unique. Requires not null.
+
+    ```groovy:no-line-numbers
+    id(String)
+    ```
+
+- `ResourceLocation`. The path to the frame texture. The resource must be loaded via external methods. Can be overridden by the recipe. [Textures that come with the mod](https://github.com/Ender-Development/selection-gui-crafting-continued/tree/master/src/main/resources/assets/selectionguicrafting/textures/gui/frame). (Default `selectionguicrafting:textures/gui/frame/default.png`).
+
+    ```groovy:no-line-numbers
+    frame(String)
+    frame(ResourceLocation)
+    ```
+
+- `ResourceLocation`. The path to the border texture. The resource must be loaded via external methods. [Textures that come with the mod](https://github.com/Ender-Development/selection-gui-crafting-continued/tree/master/src/main/resources/assets/selectionguicrafting/textures/gui/background). (Default `selectionguicrafting:textures/gui/background/default.png`).
+
+    ```groovy:no-line-numbers
+    border(String)
+    border(ResourceLocation)
+    ```
+
+- `ArrayList<GsSound>`. The sounds that will be played when the recipe is crafted. Can be overridden by the recipe.  The first `float` value is the volume, the second `float` value is the pitch of the sound.
+
+    ```groovy:no-line-numbers
+    sound(String, float, float)
+    sound(SoundEvent, float, float)
+    sound(ResourceLocation, float, float)
+    ```
+
+- `ArrayList<GsParticle>`. The particles that will be spawned when the recipe is crafted. Can be overridden by the recipe. The first `int` value is the amount of particles, the second `float` value is the speed of the particles.
```

Comment:
"The first `int` value" implies that there is a second `int`, and "the second `float` value" implies that there is a first `float`. seems like "The `int` value" would be better